### PR TITLE
Upgrade controller-runtime to v0.8.3

### PR DESCRIPTION
**Pull request by vaibhavjainwiz**
Upgrade controller-runtime to v0.8.3 and Go Lang version 1.16.
It require to integrate with operator-sdk v1.11.0

### CODE REVIEW

**Comment by vaibhavjainwiz** [comment]
@tchughesiv  Please review.

**Comment by vaibhavjainwiz** [comment]
@tchughesiv Please review.

File: go.mod [hunk 1]
```diff
@@ -1,34 +1,22 @@
...
-
-replace (
-	// OpenShift release-4.6
-	github.com/openshift/api => github.com/openshift/api v0.0.0-20200930075302-db52bc4ef99f
-	github.com/openshift/client-go => github.com/openshift/client-go v0.0.0-20200929181438-91d71ef2122c
-
-	// Pinned to kubernetes-1.19.0
-	k8s.io/api => k8s.io/api v0.19.0
-	k8s.io/apimachinery => k8s.io/apimachinery v0.19.0
-	k8s.io/client-go => k8s.io/client-go v0.19.0 // Required by prometheus-operator
```
Comment (tchughesiv) [inline go.mod:33]: is this no longer necessary with go 1.16? i haven't tested but saw something about this listed in release notes - https://go.dev/blog/go116-module-changes

Comment (vaibhavjainwiz) [reply go.mod:33]: This is not related to go 1.16. After upgrade of api versions, now every imported dependency is compatible with k8s api v0.20.4. There is no need of `replace`.

**Review by tchughesiv — APPROVED** [review]
lgtm

**Pull request merged**